ideascube/models.py

```diff
@@ -244,6 +244,14 @@ def index_strings(self):
         ('female', _('Female')),
     )
 
+    DISABILITY_CHOICES = (
+        ('visual', _('Visual')),
+        ('auditive', _('Auditive')),
+        ('physical', _('Physical')),
+        ('cognitive', _('Cognitive')),
+        ('mental', _('Mental')),
```

Comment:
My only worry with this is whether we're not using the wrong categories here, or missing some.

Of course we can always add more in the future, but for someone in a specific situation, it can be painful to have your disability missing from a list of choices, as if you didn't exist. :disappointed: 

Of course, from the height of my abled priviledge, I have no idea whether this list is correct or not. So I'm not against, just worried that we're missing something important to someone, and hoping such an omission won't cause too much pain.

Or should this be a free-form `CharField` instead?